tests/fixtures/qembot_mocks.py

```diff
@@ -0,0 +1,115 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: MIT
+
+import json
+from typing import Any, List
+from unittest.mock import patch
+
+import pytest
+
+from openqabot.types import Data
+
+
+@pytest.fixture
+def mock_incident_settings_data_852() -> List[Data]:
+    """Mock incident_settings_data with different Flavors in response"""
+    pr_number = 852
+    return [
+        Data(
+            incident=pr_number,
+            settings_id=110,
+            flavor="TestFlavor",
+            arch="x86_64",
+            distri="sle",
+            version="15-SP7",
+            build=":852:test",
+            product="SLES",
+        ),
+        Data(
+            incident=pr_number,
+            settings_id=111,
+            flavor="Server-DVD-Updates",
+            arch="aarch64",
+            distri="sle",
+            version="15-SP6",
+            build=":852:test",
+            product="SLES",
+        ),
+    ]
+
+
+@pytest.fixture
+def mock_incident_settings_data_999() -> List[Data]:
+    """Mock incident_settings_data with a single job"""
+    pr_number = 999
+    return [
+        Data(
+            incident=pr_number,
+            settings_id=200,
+            flavor="TestFlavor",
+            arch="x86_64",
+            distri="sle",
+            version="15-SP6",
+            build=":999:test",
+            product="SLES",
+        )
+    ]
+
+
+@pytest.fixture
+def qem_mock_get_incident_settings() -> Any:
```

Comment:
```suggestion
def mock_get_incident_settings() -> Any:
```

or

```suggestion
def mock_qem_get_incident_settings() -> Any:
```

The same applies to other functions. I'd still always start with `mock_` for functions which do mocking and keep the location/name of the function being mocked together.